CAIPs/caip-285.md

```diff
@@ -65,6 +66,26 @@ An example of a successful response follows:
 }
 ```
 
+### Extended Usage: partial revokes via `wallet_revokeSession`
+
+When the optional `scopes` parameter is passed, the `wallet_revokeSession` method MUST partially revoke the active session.
+
+```jsonc
+{
+  "id": 1,
+  "jsonrpc": "2.0",
+  "method": "wallet_revokeSession",
```

Comment:
@pedrouid made a good point that this doesn't map well to CRUD -- reduceSession, maybe? separate CAIP so as not to add a MUST? non blocking, just ideas to discuss at next call without a full agenda